### ISSUE TITLE
Embedded MWC Node, getting "TOR process is already running." message continuously.

### ISSUE THREAD

**Issue by Viv-dev-mwc** [mwcproject/mwc-qt-wallet#859]
**Description:**
In Mainnet QT wallet, selected "Embedded MWC Node" with the option "Don't use TOR for MWC Node".
After wallet restart and Syncing being started, getting "TOR process already running." messages continuously.
Please refer to the below screenshots for more details. 
 
![image](https://user-images.githubusercontent.com/63310656/117331300-429f1400-aeb4-11eb-8340-02b89766eb8e.png)

![image](https://user-images.githubusercontent.com/63310656/117331113-0b306780-aeb4-11eb-99a9-ad3f49ceec56.png)

**Comment by bayk** [comment]
@Viv-dev-mwc,  please provide steps to reproduce it reliably.
Such error can occasionally happens. I can't find steps to reproduce it with a current build. 

Playing with embedded node and Tor setting option doesn't make it fail.

**Comment by Viv-dev-mwc** [comment]
**Steps to follow**
1. Login to QT wallet and select "Embedded MWC Node" with the option "Don't use TOR for MWC Node".
2. Restart the wallet.
3. While sync is in-progress I'm getting "TOR process is already running." message continuously.

**Comment by bayk** [comment]
Yes, that what I do and it works fine for me. I can restart wallet without any issue.
